doc-content/enterprise-only/installation/installer-jws-cli-proc.adoc

```diff
@@ -46,7 +46,7 @@ Specify the home directory of the following servers:  Red Hat JBoss EAP 7.1 or R
 [/home/user/{PRODUCT_INIT_CAP}-{PRODUCT_VERSION_LONG}/jboss-eap-7.1]
```

Comment:
Outdated eap version in example

Reply:
The file is obsolete. Removed from repo.